grails-app/domain/mini/asaas/Payment.groovy

```diff
@@ -0,0 +1,38 @@
+package mini.asaas
+
+import mini.asaas.enums.BillingType
+import mini.asaas.enums.PaymentStatus
+import mini.asaas.utils.BaseEntity
+import java.time.LocalDate
+
+class Payment extends BaseEntity {
+
+    Payer payer
+
+    BillingType billingType
+
+    BigDecimal value
+
+    PaymentStatus status
+
+    LocalDate dueDate
+
+
+    static constraints = {
+        payer nullable: false
+        billingType nullable: false
+        value nullable: false, min: 0.01G
+        status nullable: false
+        dueDate nullable: false
+    }
+
+    static mapping = {
+        version false
+    }
+
+    @Override
+    String toString() {
+        return "Pagamento de ${value} reais para ${payer?.name} - ${status}"
+    }
```

Comment:
pq vc precisa disso aqui? 
é bom evitar criar coisas no PR que ainda não são usadas, pra facilitar o entendimento... possivelmente tem formas melhores de fazer isso que vc precisa aqui, então é sensato deixar pra fazer somente quando for usado de fato pra podermos ser mais assertivos nas soluções propostas